src/main/java/it/mulders/mcs/search/printer/clipboard/SystemClipboard.java

```diff
@@ -0,0 +1,40 @@
+package it.mulders.mcs.search.printer.clipboard;
+
+import java.awt.*;
```

Comment:
Could you please refrain from using wildcard imports?

Reply:
✅